vendor/deno-dom/element.ts

```diff
@@ -370,4 +370,18 @@ export class Element extends Node {
     }
     return null
   }
+
+  insertBefore(newNode: Node, existingNode: Node | null) {
+    if (!existingNode) {
+      return this.appendChild(newNode);
+    } else {
+      for (let i = 0; i < this.childNodes.length; i++) {
```

Comment:
I haven't used `for` loops in such a long time I feel like it isn't looping fully over the `this.childNodes`, correct me if I am wrong.

Reply:
You're doubting me, but I really think it's right.